README.md

```diff
@@ -1,12 +1,45 @@
 This is a public chat app with mock responses generated by file core/mock.py.
-- Frontend is in React and is located in /frontend directory
-- The backend is inside the /app directory
 
-# Important files:
-- .env - environmental variables for Docker container
-- docker-compose - Docker compose file to deploy docker
-- routers/chat.py - the main chat info is routed here from the front end
-- main.py - an attempt at using socket.io
+### Prerequisites
+- Docker and Docker Compose must be installed on your system.
+- A basic understanding of Docker, Docker Compose, React, and FastAPI is required.
 
-# Deployment
-To deploy as a docker run docker-compose up.
+### Project Structure Overview
+- **Frontend**: A React application located in the `/frontend` directory.
+- **Backend**: A FastAPI application located inside the `/app` directory.
+
+### Important Files
+- `.env`: Contains the environment variables required for the Docker containers.
+- `docker-compose.prod.yml`: Docker Compose file used to orchestrate the production deployment.
+- `docker-compose.dev.yml`: Docker Compose file used to orchestrate the development deployment.
+- `routers/chat.py`: Handles routing of chat information from the frontend.
+- `main.py`: Incorporates socket.io for real-time communication features.
+
+### Step-by-Step Production Deployment Guide
+
+#### 1. Preparing Environment Variables
+- Duplicate the provided `.env` sample. Customize the values to suit your specific environment and needs.
+
+#### 2. Configuring the Docker Compose File
+- Examine the `docker-compose.prod.yml` file. It outlines three services: `Traefik` for reverse proxying, `backend`, and `frontend`.
+- Adjust volume paths, environment variables, and Traefik labels to match your setup.
```

Comment:
This is very generic. at least some guidance on what labels are important, which should be edited, and when. some labels may not need editing.
Where do I find volume paths to adjust and why I need to adjust them if for previous deployments without https we did not need to adjust them as they always remained the same.

Reply:
Please check the prior step and specifically the .env file where I commented out each variable and what it is meant for